### ISSUE TITLE
Change all reference from "brackets" to "parentheses"

### ISSUE THREAD

**Issue by maomeara63** [GSA/notifications-admin#1134]

**Comment by maomeara63** [comment]
"Brackets" are how the British refer to parentheses. Change all instances of brackets to parentheses in both the app and supporting docs.

**Comment by ecayer** [comment]
@em-herrick Hi! Can you clarify what this is waiting for? Or is it just low priority?

**Comment by em-herrick** [comment]
Yes low priority and an opportune time to actually do a full replacement scan across the site.

**Comment by dmvancura** [comment]
moved this up as it could be could to consider having an accurate description of the thing encasing the customizable placeholders within a text template

**Comment by dmvancura** [comment]
marked for quarterly content review and push to prod, scoped ideally for march 2025

**Comment by dmvancura** [comment]
it is possible to move this up with your assistance in making the update, @jonathanbobel? so instructions are accurate
